sdks/python/container/license_scripts/dep_urls_py.yaml

```diff
@@ -0,0 +1,116 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+pip_dependencies:
+  # this is a list of packages whose license cannot be pulled with pip-licenses tool.
+  # pip-licenses scans site-packages/ directory to get a list of packages installed.
+  # for the packages who missing licenses in distributed package should be pulled from a url or manually added.
+  # An entry is needed if license cannot be pull with pip-licenses in ANY Python environment.
+
+  # template:
+  # dependency_name:
+  #   license: url_to_license, "skip" or "manual"
+  #     # skip: don't add license for the dependency
+  #     # manual: license is manually added. The license is copied from manual_licenses/{dep}/LICENSE.
+  #   notice: it's optional, url_to_notice
+  apache-beam:
+    license: "skip" # don't include self dependency.
+  appdirs:
+    license: "https://raw.githubusercontent.com/ActiveState/appdirs/master/LICENSE.txt"
+  avro:
+    license: "https://raw.githubusercontent.com/apache/avro/master/LICENSE.txt"
+    notice: "https://raw.githubusercontent.com/apache/avro/master/NOTICE.txt"
+  avro-python3:
+    license: "https://raw.githubusercontent.com/apache/avro/master/LICENSE.txt"
+    notice: "https://raw.githubusercontent.com/apache/avro/master/NOTICE.txt"
+  backports.weakref:
+    license: "https://raw.githubusercontent.com/PiDelport/backports.weakref/master/LICENSE"
+  chardet:
+    license: "https://raw.githubusercontent.com/chardet/chardet/master/LICENSE"
+  certifi:
+    license: "https://raw.githubusercontent.com/certifi/python-certifi/master/LICENSE"
+  cython:
+    license: "https://raw.githubusercontent.com/cython/cython/master/LICENSE.txt"
+  enum34:
+    license: "manual"
```

Comment:
Another possibility is to link the license from Beam repo, to make the treatment of all packages more uniform. One would need to check licenses in first, though.